django/core/cache/__init__.py

```diff
@@ -31,22 +31,9 @@
 
 def _create_cache(backend, **kwargs):
     try:
-        # Try to get the CACHES entry for the given backend name first
-        try:
-            conf = settings.CACHES[backend]
-        except KeyError:
-            try:
-                # Trying to import the given backend, in case it's a dotted path
-                import_string(backend)
-            except ImportError as e:
-                raise InvalidCacheBackendError("Could not find backend '%s': %s" % (
-                    backend, e))
-            location = kwargs.pop('LOCATION', '')
-            params = kwargs
-        else:
-            params = {**conf, **kwargs}
-            backend = params.pop('BACKEND')
-            location = params.pop('LOCATION', '')
+        params = {**settings.CACHES[backend], **kwargs}
```

Comment:
Can this fail with KeyError? It looks like that was transformed to InvalidCacheBackendError in the old code.

Reply:
It's used only in `CacheHandler` where we check that alias is correct: https://github.com/django/django/blob/7ec2658e1e24149f0f3244c08c361348f6ebc0e4/django/core/cache/__init__.py#L74-L77

Reply:
Maybe the separate method could be removed?

Reply:
Sounds good, I will prepare edits later today.